objects-and-methods/exercise-2/lib/trick_or_treater.rb

```diff
@@ -0,0 +1,38 @@
+require 'byebug'
+class TrickOrTreater
+  attr_accessor :bag, :sugar_level
+  def initialize(costume)
+    @costume = costume
+    @bag = Bag.new
+    @sugar_level = 0
+  end
+
+  def dressed_up_as
+    @costume.style
+  end
+
+  def has_candy?
+    if @bag == nil
+      return false 
+    else
+      !@bag.candies.empty?
+    end
+  end
```

Comment:
Add guard clause (one line if condition)

```suggestion
  def has_candy?
    return false if @bag.nil?
    
    !@bag.candies.empty?
  end
```